.sync/azure_pipelines/matrix_dependent/Ubuntu-GCC5.yml

```diff
@@ -23,6 +23,9 @@ resources:
       endpoint: microsoft
       name: microsoft/mu_devops
       ref: refs/tags/{{ sync_version.mu_devops }}
+  containers:
+    - container: linux-gcc
+      image: ghcr.io/tianocore/containers/fedora-35-build:2113a0e

```

Comment:
I think it would be nice to define the container in one place in the repo.

I planned for [.sync/Version.njk](https://github.com/microsoft/mu_devops/blob/main/.sync/Version.njk) to contain versions of various resources used in files that are synced. What do you think about adding something like a `linux_gcc_container` entry there?

The file is already imported here so you would just need to use:

```yaml
  containers:
    - container: linux-gcc
      image: {{ sync_version.linux_gcc_container }}
```

Reply:
I agree this is better. Unfortunately, the other referencing location is currently in a non file-synced file so I fear the returns may not be that great for the time being.